.gitignore

```diff
@@ -1,5 +1,7 @@
 .DS_Store
-.pyc
+*~
+*.svg
+*.pyc
```

Comment:
Maybe revert the changes to `.gitignore` since they aren't directly related to this PR?